bcauth/jinja2/account/email_confirmed.html

```diff
@@ -4,7 +4,6 @@
 {% block body_title %}{{ _("Confirm E-mail Address") }}{% endblock %}
 
 {% block content %}
-{% user_display confirmation.email_address.user as user_display %}
```

Comment:
Mmm... Ok. Given that, I vote we leave it until you get to the point where you can test this out.